website/content/en/status/report-2024-04-2024-06/clusteradm.adoc

```diff
@@ -0,0 +1,77 @@
+[.screen]
+----
+ 12.x: Regular   2, Jails   8
+ 13.x: Regular  19, Jails  68
+ 14.x: Regular  64, Jails 206
+>15.x: Regular  48, Jails   8
+-----------------------------
+Total: Regular 133, Jails 290
+Total installations:      423
+Runnning -RELEASE|{-p*}:    0
+Total geographic sites:    16
+----
+
+==== Moving cluster services to Chicago
+
+Earlier this year, we started building up our new site in Chicago.
+This quarter, we began decommissioning older machines in New Jersey and moving services to the newer machines in Chicago.
+Our long-term goal is for Chicago to become our primary location.
+This work will take several more months to complete.
+
+==== FreeBSD Official Mirrors Overview
+
+Current locations are Australia, Brazil, Germany, Japan (two full mirror sites), Malaysia, South Africa, Sweden, Taiwan, United Kingdom (full mirror site), United States of America -- California, Chicago, New Jersey (primary site), and Washington.
+
+The hardware and network connection have been generously provided by:
+
+* Cloud and SDN Laboratory at link:https://www.bbtower.co.jp/en/corporate/[BroadBand Tower, Inc]
+* link:https://www.cs.nycu.edu.tw/[Department of Computer Science, National Yang Ming Chiao Tung University]
+* link:https://deploy.equinix.com/[Equinix]
+* link:https://internet.asn.au/[Internet Association of Australia]
+* link:https://www.isc.org/[Internet Systems Consortium]
+* link:https://www.inx.net.za/[INX-ZA]
+* link:https://www.kddi-webcommunications.co.jp/english/[KDDI Web Communications Inc]
+* link:https://www.mohe.gov.my/en/services/research/myren[Malaysian Research & Education Network]
+* link:https://www.metapeer.com/[MetaPeer]
+* link:https://www.nyi.net/[New York Internet]
+* link:https://nic.br/[NIC.br]
+* link:https://www.teleservice.net/[Teleservice Skåne AB]
+* link:https://your.org/[Your.Org]
+
+New official mirrors are always welcome.
+We have noted the benefits of hosting single mirrors at Internet Exchange Points globally, as evidenced by our existing mirrors in Australia, Brazil, and South Africa.
+If you are affiliated with or know of any organizations willing to sponsor a single mirror server, please contact us.
+We are particularly interested in locations on the United States West Coast and throughout Europe.
+
+See link:https://wiki.freebsd.org/Teams/clusteradm/generic-mirror-layout[generic mirrored layout] for full mirror site specs and link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[tiny-mirror] for a single mirror site.
+
+==== Sponsorships
+
+While the day-to-day running of the FreeBSD.org cluster is overwhelmingly a volunteer team effort, larger projects (e.g. major upgrades, bringing up new cluster sites) would not be possible without sponsorship.
+
+Thanks to the link:https://freebsdfoundation.org[FreeBSD Foundation] and link:https://www.netzkommune.de[Netzkommune] for generously link:https://github.com/sponsors/ppaeps/[sponsoring Philip's time] to work on the FreeBSD.org cluster this quarter.
```

Comment:
Could you please change these lines to our standard format?

```suggestion
Sponsors: The FreeBSD Foundation and Netzkommune, for Philip's time
```

Reply:
Thanks.  Done!